src/Rules/Library/ControlShouldSupportSetInfoWPF.cs

```diff
@@ -18,7 +18,8 @@ public ControlShouldSupportSetInfoWPF()
             this.Info.Description = Descriptions.ControlShouldSupportSetInfo;
             this.Info.HowToFix = HowToFix.ControlShouldSupportSetInfo;
             this.Info.Standard = A11yCriteriaId.ObjectInformation;
-            this.Info.ErrorCode = EvaluationCode.Warning;
+            this.Info.ErrorCode = EvaluationCode.Error;
```

Comment:
@RobGallo, I changed this because you called this out as a High Impact bug, and we want to be sure and surface it. Please let me know if you disagree with this change.

Reply:
I good with this change. It's a framework issue that we want to surface. Many of the framework issues were made warnings before we had a good mechanism to surface them as such.